src/main/java/org/sagebionetworks/web/client/widget/team/JoinTeamWidget.java

```diff
@@ -164,9 +212,8 @@ public void onFailure(Throwable caught) {
 	}
 	
 	@Override
-	public void sendJoinRequest(String message, boolean isAcceptingInvite) {
+	public void sendJoinRequest(String message) {
 		this.message = message;
-		this.isAcceptingInvite = isAcceptingInvite;
```

Comment:
![image](https://cloud.githubusercontent.com/assets/8346437/9453728/36f26664-4a71-11e5-9359-0d4eef76b2cb.png)

There is also a Request Sent message at the top of the screen, but it disappears in a few seconds, so I missed it. My bad. This is for complex requests.

![image](https://cloud.githubusercontent.com/assets/8346437/9453774/7777b13a-4a71-11e5-8128-34184248d6a3.png)

If you can join with one click, whether by an invitation or by a simple request w/o authorization, it just turns into the isMemberMessage text.